src/crimson/os/store_context.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include "common/perf_counters_collection.h"
+#include "ConfigObs.h"
+
+class StoreContext {
```

Comment:
could you add a comment explaining what StoreContext is for, and how it interacts with the rest of system?

Reply:
just want StoreContext to instead of cct in this test case to get some configuration information. such as perfcounters and config.  it depends what we design in Crimson, do we need keep cct in Objectstore posix thread, or we remove cct from both seastar shard and posix thread.